test-definitions/fern/apis/response-property/definition/service.yml

```diff
@@ -0,0 +1,34 @@
+types:
+  WithDocs:
+    properties:
+      docs: string
+
+  Object:
+    properties:
+      id: string
+
+  Response:
```

Comment:
nit: would be nice to start making these examples more readable something like (e.g. naming the `Resource` something else not `Object`)